### DISCUSSION TITLE
ci.eclipse.org/mylyn now use the Organisation Folder

### DISCUSSION THREAD

**Discussion by BeckerFrank** [eclipse-mylyn/.github#21] [Announcements]
Hi,

I have disabled the Jenkins Job github-mylyn and create a new GitHub Organisation folder Job GitHub-mylyn-Orga so that changes from the two active Repositories
- org.eclipse.mylyn
- org.eclipse.mylyn.docs

are now trigger new builds and we now longer need to pull very 5 Minutes.

@akurtakov: I disable the Job github-mylyn-docs so that you have no two parallel builds.

**Comment by akurtakov** [comment]
As long as everything still works - even better.

**Reply by BeckerFrank** [reply]
With my first test I found that we need help from the Help Desk (see https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/2819).
Maybe we need to do some additional setup first, so I restore the old two jobs for now.

**Comment by wimjongman** [comment]
Thanks, Frank!

**Comment by BeckerFrank** [comment]
Now after  Pawel Stankiewicz add the missing Webhooks I turned the new Job on and disable the two other Jobs.
